internal/pkg/plugin/argocd/argocd.go

```diff
@@ -8,6 +8,7 @@ import (
 
 var defaultHelmConfig = helm.Options{
 	Chart: helmCommon.Chart{
+		ChartPath:   "",
```

Comment:
`""` is zero value of `string`, so it is unnecessary to set the default value here.

But on the other hand, you can write in this way to emphasize it.

Reply:
yes, emphasize only